Python Script to Plot Sagemaker Endpoint Utilization Metrics , Heatmap to visualize Cost for Various RPMs

dheerajoruganty opened the pull request:
*Description of changes:*

 This pull request adds a new python script to generate visualization plots for utilization metrics.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

aarora79 reviewed (changes requested):
1/ Change the order of imports to look like this
```
import logging
import pandas as pd
import seaborn as sns
import matplotlib.pyplot as plt
```

2/ Take the input as  a `pd.DataFrame` rather than filename.

dheerajoruganty commented:
Updated PR,

New changes:

Added sagemaker_cost_RPM_plot.py script to create a heatmap which visualizes Cost for various requests per minutes. Also added logging.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
